test_regress/t/t_cover_branch_option.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
```

Comment:
For all these new tests, change to t_vlcov_opt_branch.py etc and just use the existing coverage input data file that is in test directory. (You can update that data file if need to).  Much less test code and execution time then.